tests/installation/addon_products_sle.pm

```diff
@@ -128,6 +128,7 @@ sub run {
         set_var('SKIP_INSTALLER_SCREEN', 0);
     }
     $self->process_unsigned_files([qw(inst-addon addon-products)]);
+    wait_still_screen;
```

Comment:
as discussed: a `wait_still_screen` has a default stilltime of 7 seconds, means at least 7-8 seconds are wasted regardless of the result which I consider very wasteful, especially considering that this module is used in all SLE installation tests.